avm/res/db-for-my-sql/flexible-server/main.bicep

```diff
@@ -171,6 +171,9 @@ param databases array = []
 @description('Optional. The firewall rules to create in the MySQL flexible server.')
 param firewallRules array = []
 
+@description('Optional. The configurations to create in the server.')
+param configurations array = []
```

Comment:
Would recommend to add a type for the new parameter while you're at it it will be required for releasing the module as 1.0.

To do so, just add the following type to the `Definitions` section at the bottom

```bicep
param configurations configurationType[]?

@export()
@description('The type for a configuration)
type configurationType = {
  @description('Required. The name of the configuration.')
  name: string

  @description('Optional. Source of the configuration.')
  param: source ('system-default' | 'user-override')?

  @description('Optional. Value of the configuration.')
  value: string/int? // <= as per my other comment
}
```

Reply:
On a different note: Is it actually possible to create a configuration without any values? I don't have an instance deployed to confirm, but just wanted to double-check as it stikes be a bit odd that only the `name` is required

Reply:
What are the sources you compaired?